src/services/i18n.ts

```diff
@@ -32,4 +32,10 @@ i18n
     }
   })
 
+export const getNextLanguage = () => {
+  const langIndex = ENABLED_LANGUAGES.indexOf(i18n.language)
```

Comment:
if you wanna get the actual current language limited to our whitelist, you should use `languages[0]` instead --> that way, you'll get for instance `en` instead of `en-US, en-CA, en-UK, etc`

Reply:
i18n.language won't return `en` `fr` etc unless specified through query string so, wouldn't use that

Reply:
if you switch language, then languages[0] is also updated

Reply:
I'd originally built on it, but was getting random index order. It's a good point, though. I'll just sort.

Reply:
Oh, that's weird, sorting / index definitely worked for me given that's how the language switcher works

Reply:
just extracted before usage 👍